tests/components/bbox/test_device_tracker.py

```diff
@@ -0,0 +1,174 @@
+"""Tests for the Bbox device tracker platform."""
+
+from datetime import timedelta
+from unittest.mock import MagicMock
+
+from freezegun.api import FrozenDateTimeFactory
+import requests
+
+from homeassistant.components.bbox.device_tracker import (
+    MIN_TIME_BETWEEN_SCANS,
+    PLATFORM_SCHEMA,
+    get_scanner,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+
+async def test_get_scanner_success(
+    hass: HomeAssistant,
+    device_tracker_config: ConfigType,
+    mock_bbox_api: MagicMock,
+) -> None:
+    """Test successful scanner initialization."""
+    scanner = get_scanner(hass, device_tracker_config)
+    assert scanner is not None
+    assert scanner.success_init is True
```

Comment:
instead, we should let HA setup the integration with the YAML config, and then we can check if the device tracker entities that we expect are registered